Formula/blucat.rb

```diff
@@ -1,12 +1,12 @@
 class Blucat < Formula
   desc "netcat for Bluetooth"
   homepage "http://blucat.sourceforge.net/blucat/"
-  url "http://blucat.sourceforge.net/blucat/wp-content/uploads/blucat-aa3e02.zip"
-  version "0.9"
-  sha256 "6dcd6bf538a06c2f29d21a9e94d859d91667a7014244462bffca9767bba5307d"
+  url "https://github.com/ieee8023/blucat/archive/v0.91.tar.gz"
+  sha256 "3e006d25b7e82689826c89ffbbfa818f8b78cced47e6d0647e901578d330a2f4"
+  head "https://github.com/ieee8023/blucat.git"
 
   depends_on "ant" => :build
-  depends_on :java => "1.6"
+  depends_on :java
```

Comment:
iMac-TMP:~ joe$ for i in 1.4 1.5 1.6 1.7 1.8 1.9 ; do printf "$i: "; /usr/libexec/java_home --version $i &>/dev/null ; echo $? ; done
1.4: 2
1.5: 2
1.6: 0
1.7: 2
1.8: 0
1.9: 2
iMac-TMP:~ joe$